nixos/maintainers/scripts/ec2/amazon-image.nix

```diff
@@ -61,6 +63,9 @@ in {
           ${optionalString config.ec2.hvm ''
             ec2.hvm = true;
           ''}
+          ${optionalString config.ec2.efi ''
+            ec2.efi = true;
```

Comment:
Implemented in the same pattern as `ec2.hvm`, but I don't know the reason for this.